go/kbfs/libkbfs/prefetcher_test.go

```diff
@@ -103,8 +103,19 @@ func initPrefetcherTest(t *testing.T) (*blockRetrievalQueue,
 	return initPrefetcherTestWithDiskCache(t, nil)
 }
 
-func shutdownPrefetcherTest(q *blockRetrievalQueue) {
-	q.Shutdown()
+func shutdownPrefetcherTest(t *testing.T, q *blockRetrievalQueue, syncCh chan struct{}) {
```

Comment:
You're not actually using `t` in this function, but it's harmless.  Maybe it could be improved by passing in `ctx` too, and then wrapping a `select` around the final `<-ch` along with a `<-ctx.Done()`, and failing using `t` if the context times out.  Not a big deal though.

Reply:
Yeah originally I had it using `t` but then refactored a bunch to what it is now.

Reply:
I'll do what you suggested.